tests/dist/src/test/java/org/hawkular/apm/tests/dist/AnalyticsServiceITest.java

```diff
@@ -108,7 +109,7 @@ public void initTest() {
     public void testGetUnboundEndpoints() throws Exception {
         Trace trace1 = new Trace();
         trace1.setId("1");
-        trace1.setStartTime(System.currentTimeMillis() - 4000); // Within last hour
+        trace1.setStartTime(TimeUnit.MILLISECONDS.toMicros(System.currentTimeMillis()) - 4000); // Within last hour
```

Comment:
Yes know it was my comment, but don't know what I was trying to say :) Possibly was just setting a value to override the default 1hour - but either way any time data needs to be converted to microseconds to be consistent with overall change.